bin/index.toit

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2025 Toit contributors
+// Use of this source code is governed by an MIT-style license that can be
+// found in the LICENSE file.
+
+import io
+
+import .utils
+
+MAX-ENTRIES_ := 15
+
+build-entries_ index/Map --transcripts-dir/string -> string:
+  buffer := io.Buffer
+  entry-count := 0
+  index.do --values: | entry/Map |
+    url := "$transcripts-dir/$entry["filename"]"
+    display := entry["displayName"]
+    escaped-display := html-encode display
+    buffer.write """
+    <li>
+      <a href="url">$display</a>
+    </li>
+    """
+    if entry-count > MAX-ENTRIES_:
```

Comment:
Maybe you need to increase `entry-count` somewhere?

Reply:
oops.
done.